### ISSUE TITLE
Filtering within

### ISSUE THREAD

**Issue by cpick20** [dbeaver/dbeaver#13020]
**Is your feature request related to a problem? Please describe.**
In DBeaver filtering columns is not very useful since other columns do not filter within that condition.

**Describe the solution you'd like**
I think it would be valuable if when you filtered a column in DBeaver the other columns were filtered in as well. So if I filtered Column1 based on a variable value, the other columns filter option should only show the options of values within that filter.

**Comment by shirotakakah** [comment]
I agree with this feature request.  This feature would be very beneficial in my work.

**Comment by uslss** [comment]
thank you for the idea
